tests/gitserver/Dockerfile

```diff
@@ -94,13 +114,29 @@ RUN for i in $(seq 1 $REPO_COUNT); do \
     git init && \
     git config --global user.email "you@example.com" && \
     git config --global user.name "Your Name" && \
-    echo "some: data" > data.yaml && \
+    echo "apiVersion: apps/v1\nkind: Deployment\nmetadata:\n  name: nginx-deployment\nspec:\n  selector:\n    matchLabels:\n      app: nginx\n  replicas: 2\n  template:\n    metadata:\n      labels:\n        app: nginx\n    spec:\n      containers:\n      - name: nginx\n        image: nginx:1.14.2\n        ports:\n        - containerPort: 80" > data.yaml && \
```

Comment:
If the BDD test need to verify syncing a Numaflow pipelines, this needs to be updated to the  pipelines/controller/isbs spec? It seems to be a bit rigid.

Reply:
yeah bdd tests dont use this ,i will update the pr

Reply:
Sounds good, move to draft for now.